lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -74,6 +87,7 @@ pub struct Client {
     write_stream: Option<RdpWriteStream>,
     function_receiver: Option<FunctionReceiver>,
     x224_processor: Arc<Mutex<x224::Processor>>,
+    resize_withholder: Arc<Mutex<ResizeWithholder>>,
```

Comment:
Maybe call it `pending_resize`? 
And maybe we can go with plain `Arc<Mutex<Option<(u32,u32)>>>`, or 
`type PendingResize = Option<(u32,u32)>; .... Arc<Mutex<PendingResize>>`